pyproject.toml

```diff
@@ -34,7 +34,7 @@ boto3-stubs = {extras = ["bedrock", "iam", "opensearch", "s3", "sagemaker"], ver
 
 # drivers
 cohere = { version = ">=4", optional = true }
-anthropic = { version = "^0.7.8", optional = true }
+anthropic = { version = "^0.17.0", optional = true }
```

Comment:
Lets bump this up to be in sync with the [other PR](https://github.com/griptape-ai/griptape/pull/694/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R37)